acme/src/main/java/io/micronaut/acme/services/AcmeService.java

```diff
@@ -482,7 +507,7 @@ private void doChallengeSpecificSetup(Authorization auth, Challenge challenge) t
      * Setup the certificate that has been saved to disk and configures it for use.
      */
     public void setupCurrentCertificate() {
-        eventPublisher.publishEvent(new CertificateEvent(getCurrentCertificate(), getDomainKeyPair(), false));
+        eventPublisher.publishEvent(new CertificateEvent(getDomainKeyPair(), false, getFullCertificateChain()));
```

Comment:
if getFullCertificateChain returns an empty array, we should publish a the event. Since `CertificateEvent`  assumes a non empty array due to `CertificateEvent::getCert` where it gets the first certificate in the array. should we?

Reply:
Resolved via https://github.com/micronaut-projects/micronaut-acme/pull/161/commits/6a5dd4466aadfa3ee97dec776869d8940a036cdc